android/src/main/kotlin/com/abilia/system_settings_editor/SystemSettingsEditorPlugin.kt

```diff
@@ -112,6 +114,35 @@ class SystemSettingsEditorPlugin: FlutterPlugin, MethodCallHandler, ActivityAwar
     return 255
   }
 
+  private fun setSoundEffectsHandler(call: MethodCall, result: Result) {
+    val enabled: Boolean? = call.argument("sound_effects_enabled")
```

Comment:
I'd like to have the arguments as camel case. Seems to be default for other plugins. And I guess also for json.

Reply:
I just think it's good to have a working example in the example app so this is good enough for me.